doc/sphinx/src/governance.rst

```diff
@@ -0,0 +1,138 @@
+Organizational Structure
+==========================
+
+| ``Phoebus`` uses a *hybrid flat-and-hierarchical* organizational
+  structure.
+| The flat structure fosters power sharing among senior and junior
+  members and avoids the many pitfalls of power-over dynamics. For
+  example, in power-over structures, someone with explicit power can
+  inadvertently, or on rare occasions intentionally, silence critical
+  voices.
+| This dynamic can quickly lead to an organization overlooking inherent
+  problems that could lead to ineffectiveness or even catastrophic
+  failure of the team.
+
+| In contrast, our aim is to illuminate potential problems early and
+  quickly find solutions.
+| This strategy also efficiently taps into the diverse wealth of
+  knowledge of the team.
+| At the same time, the PI and Co-PI have financial and reporting
+  responsibilities to their respective universities and funding
+  agencies.
+| The general procedure for the hybrid structure is as follows.
+| The flat structure will make major decisions in direction and
+  resources, and the PI and Co-PI will assume the responsibility for
+  executing those decisions.
+| The following gives details on this hybrid structure.
+
+Team
+----
+
+The founding team consists of the following: original core developers
+and Maintainers at Los Alamos National Lab, Jeremiah Murphy at Florida
+State University, and Carl Fields at University of Arizona.
+
+Since, an objective of Phoebus is to grow the developer and user base,
+the Phoebus Project will have an explicit mechanism for adding new
+members. There are three categories for ``Phoebus`` members:
+maintainers, contributors, and users.
+
+**User**: As open-source software, anyone may download, compile, and
+execute ``Phoebus``. New users will encouraged to join communication
+channels for connection with the larger ``Phoebus`` community.
+
+**Contributor**: Users who wish to contribute to ``Phoebus`` may do so
+following our Development Model for new users. These changes will be
+integrated into the main branch of the code upon approval. Consistent
+contributions also provides the opportunity for Contributors to be
+eligible to join as a *Maintainer*, the final approval of which will
+require a vote.
+
+**Maintainer**: These are the core team who have “merge” permissions on
+the repo and are working to keep the code sustainable long term.
+
+To maintain their active status as a User, Contributor, or Maintainer,
+all members will agree to adhere to a Code of Conduct.
+
+
+.. _mou:
+
+Memorandum of Understanding
+---------------------------
+
+In addition to our Community Code of Conduct, all code Maintainers will
+sign a memorandum of understanding (MoU) that will describe general
```

Comment:
Sorry to keep picking on this (and dropping in on something that has apparently already been discussed elsewehere) but this would be another code of conduct I would be asked to sign to retain e.g. my merge privileges, and I'm not comfortable with that model for a code I've already been working on for years.

The below are reasonable suggestions, but I would ask that this language be converted from signing an MoU to something like, "We ask that maintainers especially keep the following suggestions in mind to help facilitate an efficient and welcoming collaboration". With the LANL COC as a backdrop and this repository being hosted on the LANL github I think we have robust protection against bad actors.

There's also another issue here: astro funding is intermittent at LANL, and people like me may not always have the time and funding to be regular attendees of monthly meetings. But at the same time I wouldn't want my merge privileges to come and go depending on whether I am currently in active developer mode, I think that adds unnecessary friction. Maybe there is a two-tiered approach here, where contributors actively involved in feature development are encouraged to organize/attend a monthly meeting, and changes to the code broader than development of specific features should be submitted to a broader set of maintainers (for example through tagging more people on PRs, or in a github issue proposing a substantial code change).

Reply:
I completely agree with everything here. I've already changed the language away from "signing," (although I clearly missed a spot) but I like the suggested language more. I've also added an addendum that "meeting cadence and attendance are subject to activity." I certainly don't want to create any friction or revoke any privileges. I'll iterate on these things today. Thanks for your thoughts and sorry that this came out of nowhere.

Reply:
FWIW, this concern was brought up and addressed in the proposal. These docs got out of sync on a few key points. This is one of them. I agree with joining when you can for core developers.